charts/tempo-distributed/values.yaml

```diff
@@ -142,6 +142,93 @@ ingester:
+    targetCPUUtilizationPercentage: 60
+    # -- Target memory utilisation percentage for the metrics-generator
+    targetMemoryUtilizationPercentage:
+  image:
+    # -- The Docker registry for the metrics-generator image. Overrides `tempo.image.registry`
+    registry: null
+    # -- Docker image repository for the metrics-generator image. Overrides `tempo.image.repository`
+    repository: null
+    # -- Docker image tag for the metrics-generator image. Overrides `tempo.image.tag`
+    tag: null
+  # -- The name of the PriorityClass for metrics-generator pods
+  priorityClassName: null
+  # -- Labels for metrics-generator pods
+  podLabels: {}
+  # -- Annotations for metrics-generator pods
+  podAnnotations: {}
+  # -- Additional CLI args for the metrics-generator
+  extraArgs: []
+  # -- Environment variables to add to the metrics-generator pods
+  extraEnv: []
+  # -- Environment variables from secrets or configmaps to add to the metrics-generator pods
+  extraEnvFrom: []
+  # -- Resource requests and limits for the metrics-generator
+  resources: {}
+  # -- Grace period to allow the metrics-generator to shutdown before it is killed. Especially for the ingestor,
+  # this must be increased. It must be long enough so metrics-generators can be gracefully shutdown flushing/transferring
+  # all data and to successfully leave the member ring on shutdown.
+  terminationGracePeriodSeconds: 300
+  # -- Affinity for metrics-generator pods. Passed through `tpl` and, thus, to be configured as string
+  # @default -- Hard node and soft zone anti-affinity
+  affinity: |
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchLabels:
+              {{- include "tempo.metricsGeneratorSelectorLabels" . | nindent 10 }}
+          topologyKey: kubernetes.io/hostname
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchLabels:
+                {{- include "tempo.metricsGeneratorSelectorLabels" . | nindent 12 }}
+            topologyKey: failure-domain.beta.kubernetes.io/zone
+  # -- Node selector for metrics-generator pods
+  nodeSelector: {}
+  # -- Tolerations for metrics-generator pods
+  tolerations: []
+  # -- Extra volumes for metrics-generator pods
+  extraVolumeMounts: []
+  # -- Extra volumes for metrics-generator deployment
+  extraVolumes: []
+  persistence:
+    # -- Enable creating PVCs which is required when using boltdb-shipper
+    enabled: false
+    # -- Size of persistent disk
+    size: 10Gi
+    # -- Storage class to be used.
+    # If defined, storageClassName: <storageClass>.
+    # If set to "-", storageClassName: "", which disables dynamic provisioning.
+    # If empty or set to null, no storageClassName spec is
+    # set, choosing the default provisioner (gp2 on AWS, standard on GKE, AWS, and OpenStack).
+    storageClass: null
+  config:
```

Comment:
Can we add somewhere overrides? It is required for metric-generator making to work, if I'm not mistaken. (Correct me if I'm wrong)

`global_overrides:
  metrics_generator_processors: 
  - service-graphs
  - span-metrics
`

Reply:
i added a comment.